.github/workflows/publish-to-dockerhub.yml

```diff
@@ -0,0 +1,79 @@
+
+name: Publish Docker image
+
+on:
+  push:
+    branches: build-docker-images
+  release:
+    types: [published]
+  # Allow workflow to be triggered manually.
+  workflow_dispatch:
+
+jobs:
+  push_sbtc_to_registry:
+    name: Push sBTC Docker Image to Docker Hub
+    runs-on: ubuntu-latest
+    steps:
+      - name: Check out the repo
+        uses: actions/checkout@v4
+      - name: Set up QEMU
+        uses: docker/setup-qemu-action@v2
+      - name: Setup Docker Buildx
+        uses: docker/setup-buildx-action@v3
+      - name: Log in to Docker Hub
+        uses: docker/login-action@v3
+        with:
+          username: ${{ secrets.DOCKERHUB_USERNAME }}
+          password: ${{ secrets.DOCKERHUB_PASSWORD }}
+      - name: Build and push Docker image
+        uses: docker/build-push-action@v5
+        with:
+          # We use ubuntu-latest which implies amd64
+          context: "{{ defaultContext }}"
+          file: ./devenv/sbtc/docker/Dockerfile
+          # TODO: Set up docker publishing to use the tag name when publishing and to do
+          # something else when triggered manually. $\{{ github.event.release.tag_name }}
+          tags: blockstack/sbtc:sbtc
```

Comment:
It should be explicit it is Devenv's image.

Reply:
One note here is that `sbtc` is the dockerhub repostiory with all the devnev images, and the following `sbtc` is the tag for the actual sbtc image.

What might be better, is if we changed each image to have the name `blockstack/sbtc:devenv-[service]` so that it's clear this is for the devenv. Thoughts?

sbtc dockerhub repo: https://hub.docker.com/r/blockstack/sbtc